phy/utils/_misc.py

```diff
@@ -98,3 +100,17 @@ def _show_shortcuts(shortcuts, name=''):
     for name in sorted(shortcuts):
         print('{0:<40}: {1:s}'.format(name, _show_shortcut(shortcuts[name])))
     print()
+
+def _git_version():
```

Comment:
ok! but this does mean that all the tests will fail if someone doesn't have git installed, or have downloaded the package rather than cloning the repository (even if they do have git installed).